sqlx-core/src/postgres/types/time.rs

```diff
@@ -252,6 +256,40 @@ impl Encode<Postgres> for OffsetDateTime {
     }
 }
 
+impl TryInto<Duration> for PgInterval {
```

Comment:
The chrono and time files are already pretty big. This impls might be better in `interval.rs` with the appropriate `cfg` flags. Just a minor thought.